modules/aruco/src/aruco.cpp

```diff
@@ -667,14 +704,8 @@ static void _identifyCandidates(InputArray _image, vector< vector< vector< Point
     int ncandidates = (int)_candidatesSet[0].size();
     vector< vector< Point2f > > accepted;
     vector< vector< Point2f > > rejected;
-
     vector< vector< Point > > contours;
 
-    CV_Assert(_image.getMat().total() != 0);
```

Comment:
this condition was checked already (in detectMarkers)

Reply:
assumption checks are necessary, see above

Reply:
fixed